Comanche055/ASSEMBLY_AND_OPERATION_INFORMATION.agc

```diff
@@ -865,15 +860,15 @@
 # -VV-
 # POSITION8		XXXX.X NAUT MI		DP	BIT 1 OF LOW REGISTER =
 #									 -28
-#							4 x 6,373,338 x 2
+#							4 X 6,373,338 X 2
 #							--------------------
-#								1852
+#							      1852
 #							NAUT MI.
 #
 # -XX-
-# POSITION 9		XXX.XX NAUT MI		DP	BIT 1 OF LOW REGISTER =
```

Comment:
That sounds about right: "took the existing Colossus 249 source-code files,
or on occasion the Luminary 131 source-code files, and laboriously compared
them line-by-line to the Comanche 055 assembly-listing page images, porting
the differences they found."